### DISCUSSION TITLE
How to install mopac 2022 in Rocky (RHL) 9

### DISCUSSION THREAD

**Discussion by johnwkeller54** [openmopac/mopac#113] [Q&A] [answered]
I am testing computational chem programs in Rocky Linux 9 (derived from Red Hat Linux 9). MOPAC2016 installs and runs fine, as do G16 and ORCA5. However, the mopac-22.0.4-linux.run script gives an error message and random output. Qt5 and Qt5-qtx11 have been installed, based only on the reference to Qt elsewhere on this site. Rocky Linux 9 uses the Wayland window system by default rather than X11 - could that be the problem?
John Keller
University of Alaska Fairbanks

**Comment by johnwkeller54** [comment]
Here is a screenshot of the terminal output mentioned above.
![Screenshot from 2022-09-04 11-59-39](https://user-images.githubusercontent.com/112823030/188331537-109085ef-54a1-4464-9481-8f17f932c81e.png)

**Comment by susilehtola** [comment, marked as answer]
MOPAC is part of Fedora and Fedora EPEL. I just recently released the RHEL 9 packages in https://bodhi.fedoraproject.org/updates/FEDORA-EPEL-2022-4e3e09c5d8

If EPEL is already enabled on your system, all you need to do is
```# dnf install mopac```
Instructions for enabling the EPEL repositories can be found at https://docs.fedoraproject.org/en-US/epel/#_el9

**Comment by johnwkeller54** [comment]
Thank you. Mopac installed on the Rocky Linux 9 system. I finally found (the executable?) in /usr/bin directory. When I tried to run it from the cml ("/usr/bin/mopac h2o.mop " ), a message was shown on the terminal "MOPAC Job: "h2o.mop" ended normally...:. but no output could be seen. How do you run mopac installed in this fashion?

**Reply by susilehtola** [reply]
The output is probably in `h2o.out`.

**Comment by godotalgorithm** [comment]
I may be overestimating and overstating the support of QT-based installers by major Linux distributions - I will revise language on the website accordingly with the next release. As Susi noted, MOPAC is now available through Linux package managers (Fedora and Conda so far), so incompatibilities with QT should not be a barrier for easy access to MOPAC.

The standard output file for MOPAC is `name.out` when the input file is named `name`, `name.mop`, `name.dat`, `name.arc`, or `name.txt`. The output files are written to the same directory as the input file.
